packages/yourloops/pages/hcp/patients/team-code-dialog.tsx

```diff
@@ -83,15 +79,15 @@ function TeamCodeDialog(props: TeamCodeDialogProps): JSX.Element {
   const buttonsClasses = makeButtonsClasses();
   const classes = dialogClasses();
   const { code, name, onClose } = props;
-  const dialogIsOpen = code.match(reCode) !== null;
-
-  let displayCode = "";
-  if (dialogIsOpen) {
-    displayCode = code.replace(reCode, "$1 - $2 - $3");
-  }
+  const dialogIsOpen = !!code.match(reCode);
```

Comment:
Not sure if theses changes was necessary

Reply:
I put the code on a diet 😅